docs/installation/community-operator.md

```diff
@@ -15,24 +25,31 @@ Before you start make sure you have a OCP 4.13 cluster running.
 ![](../fig/ocp_installation/operator_installation_ocp_2.png)
 > Note: The current OCP console may display a JavaScript error (expected to be fixed in 4.13.5), but it doesn't affect the rest of the steps. Fixes are currently available on the 4.13.0-0.nightly-2023-07-08-165124 build of OCP console.
 
-
+---
 ## Installing the Grafana operator
 
-- Get the API Bearer Token by executing the following `oc` command:
-```sh
-oc whoami --show-token
-sha256~RQOIZC73Hc5pabivSp648Hkke5R0dC45D-7u-RLHVYM
-```
-- Add the token from `step 1` in `hack/dashboard/openshift/03-grafanadatasource-UPDATETHIS.yaml`
-![](../fig/ocp_installation/operator_installation_ocp_3.png)
+### Deploy the Grafana Operator
+
+The current API Bearer token needs to be updated in the `GrafanaDataSource` manifest so that the `GrafanaDataSource` can authenticate to the Prometheus. The following command will update the manifest and deploy Grafana Operator in the namespace `kepler-operator-system`
 
-- Deploy the Grafana Operator by running the following command:
 ```sh
-$ hack/dashboard/openshift/deploy-grafana.sh
+BEARER_TOKEN=$(oc whoami --show-token)
+hack/dashboard/openshift/deploy-grafana.shhack/dashboard/openshift/deploy-grafana.sh
```

Comment:
```suggestion
hack/dashboard/openshift/deploy-grafana.sh
```

Reply:
Also like @stleerh already pointed out - https://github.com/sustainable-computing-io/kepler-operator/blob/0ce4d2dd7ad971dc18866878e5aa9b8570db80ea/hack/dashboard/openshift/deploy-grafana.sh#L30 does indeed create a BEARER_TOKEN for grafana SA but I am dubious if that script really works